src/fh-shared-kernel.shared-kernel/Identity/AppStart/ServiceCollectionExtension.cs

```diff
@@ -24,9 +24,16 @@ public static class ServiceCollectionExtension
         /// Configures UI to authenticate using Gov Login
         /// </summary>
         public static void AddAndConfigureGovUkAuthentication(
-            this IServiceCollection services, IConfiguration configuration)
+            this IServiceCollection services, ConfigurationManager configuration)
         {
             services.AddHttpContextAccessor();
+
+            var useStubAuthentication = configuration.GetValue<bool>("GovUkOidcConfiguration:StubAuthentication:UseStubAuthentication");
+            if (useStubAuthentication)
+            {
+                configuration.AddJsonFile("stubUsers.json", true);
+            }
+
             services.Configure<GovUkOidcConfiguration>(configuration.GetSection(nameof(GovUkOidcConfiguration)));
 
             var config = configuration.GetGovUkOidcConfiguration(); 
```

Comment:
move line 39(var config = configuration.GetGovUkOidcConfiguration();) above line 30 and then you can use POCO instead of 

var useStubAuthentication = configuration.GetValue<bool>("GovUkOidcConfiguration:StubAuthentication:UseStubAuthentication");
            if (useStubAuthentication)
i.e.
 if (config.StubAuthentication.UseStubAuthentication)
 {